.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+ignore=W503,E501
```

Comment:
no max line length ?

Reply:
Black formats lines with a soft limit of 88 characters, but sometimes will produce lines slightly longer than that. I've found it best to let Black handle all formatting questions, including line length, and let flake8 focus on structural concerns.